lib/tools/adb-emu-commands.js

```diff
@@ -5,42 +5,46 @@ import path from 'path';
 
 let emuMethods = {};
 
-emuMethods.isEmulatorConnected = async function (udid) {
-  let emulators = await this.getConnectedEmulators();
-  if (!emulators.length) {
-    return false;
+emuMethods.getEmulators = async function() {
```

Comment:
@jlipps I would use debounce then. It's quite unlikely for an emulator to disconnect but It might happen on large tests. I would use `debounce` with 5 minutes. what do you guys think?